mavros_msgs/msg/HilGPS.msg

```diff
@@ -0,0 +1,17 @@
+# HilControls.msg
+#
+# ROS representation of MAVLink HIL_GPS
+# See mavlink message documentation here:
+# https://pixhawk.ethz.ch/mavlink/#HIL_GPS
+
+std_msgs/Header header
+uint8 fix_type
+sensor_msgs/NavSatFix fix
```

Comment:
`NavSatFix`? You only uses lat/long/alt triplet. And since all other fields uses same raw types as mavlink it is more logical.
Also fix header.